consensus/state.go

```diff
@@ -729,7 +756,8 @@ func (cs *ConsensusState) EnterPrevoteWait(height int, round int) {
 
 	// After `timeoutPrevote0+timeoutPrevoteDelta*round`, EnterPrecommit()
 	go func() {
-		time.Sleep(timeoutPrevote0 + timeoutPrevote0*time.Duration(round))
+		time.Sleep(timeoutPrevote0 + timeoutPrevoteDelta*time.Duration(round))
+		cs.timeoutChan <- TimeoutEvent{RoundStepPrevote, height, round}
 		cs.EnterPrecommit(height, round)
```

Comment:
Wait, do we even need timeoutChan?  EnterPrecommit() can tell that there was no Polka, which implies that the PrevoteStep timed out.  Likewise EnterPrevote() can tell that the proposal wasn't complete, which implies that the ProposeStep timed out.